### ISSUE TITLE
Issues archiving "out" folder into .ktsl2stbin file

### ISSUE THREAD

**Issue by PecakePie** [Hairo/kvs-tools#2]
Hello! I'm very new to GitHub and new at modding, so if these are dumb questions please forgive me haha!

I was trying to mod the music in AOT 2: Final Battle, but when the .ktsl2stbin file is archived and renamed to BGM, there is no music in the game. Sound effects and everything else seems to work fine, but no music. There was some mention of the latest update of kvs_archive.py being broken in in the ReadMe. I was wondering how to fix this or if there is a different script that archives files into .ktsl2stbin files that I could use instead?

Thanks for the help!

**Comment by Hairo** [comment]
Yes, archiving doesn't work after the DLC, haven't looked into it since i didn't get it

**Comment by GuideToRaZe** [comment]
Hi! i'm also new to github and i'm trying to mod the bgm of AOT Wings of Freedom, it didn't work, i think i need a 1.0 version without dlc or patches, but i didn't find nothing yet. Any ideas?

thanks
